src/parts/MovieForm/MovieForm.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Row from '../../components/Skeleton/Row';
+import Col from '../../components/Skeleton/Col';
+import InputField from '../../components/forms/InputField';
+import Button from '../../components/Button/Button';
+import MultipleSelect from '../../components/forms/MultipleSelect/MultipleSelect';
+import MultipleSelectOption from '../../components/forms/MultipleSelect/MultipleSelectOption';
+import './MovieForm.css';
+
+const MovieForm = props => {
+  const [formData, setFormData] = useState(props.formData);
+
+  const addGenre = genre => {
+    const { genres } = formData;
+    if (!genres.includes(genre)) {
+      return genres.concat(genre);
+    } else {
+      return genres.filter(g => g !== genre);
+    }
+
+  };
+
+  const handleOnChange = event => {
+    const { target: { name, value } } = event;
+    const updatedFormData = {
+      ...formData,
+      [name]: name === 'genres' ? addGenre(value) : value
+    };
+    setFormData({ ...updatedFormData });
+  };
+
+  const handleOnSubmit = event => {
+    event.preventDefault();
+    props.onSubmit();
+  };
+
+  const resetForm = () => {
+    setFormData({ ...props.formData });
+  };
+
+  const { id, title, release_date, poster_path, genres, overview, runtime } = formData;
+  const { formTitle } = props;
```

Comment:
komponens fejlécben ki lehet destructure-ölni

Reply:
Fixed in [0cd3b29](https://github.com/rkasza/netflix-roulette/pull/5/commits/0cd3b297078b39a413f47a5afd50d0985f364f48)